ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandRunner.java

```diff
@@ -164,14 +166,16 @@ void fetchAndRunCommands() {
   }
 
   private void executeStatement(final QueuedCommand queuedCommand) {
-    log.info("Executing statement: " + queuedCommand.getCommand().getStatement());
+    final String encodedCommandId = Base64.getEncoder().encodeToString(
```

Comment:
Why is this being encoded? It looks to be unencoded elsewhere later on in the PR.

Reply:
Also, this behavior will be different on newer versions (7.0.x+), right? (Just checking.)

Reply:
Yes, we still have to decide the behavior on newer versions -- probably CommandId would work.

Regarding the encoding I was following our hotfix logic: https://github.com/confluentinc/ksql/commit/f60c93ed1cec86dede7a7beab13a04ff833138c6#diff-3a11b2badb0b2bc26e8e0f08fd3938ede511b9ae2ae6db18b8e3212c86d982d2R343

Reply:
Is there context for why we decided to encode the command ID in that PR anywhere? It seems odd because it's easily decoded, so it doesn't offer any real benefit, and only serves to make the log message harder to read.

Also, the error log in StatementExecutor.java in this PR logs the unencoded command ID, so I think we should be consistent. (If we want to update both to be encoded that's fine even though I don't see the benefit.)